xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -13,8 +13,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary> An abstraction representing various text encoders.</summary>
```

Comment:
An abstraction representing various text encoders --> The base class of web encoders.